pip/commands/run.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from pip.basecommand import Command, SUCCESS
+
+from pip._vendor import rwt
+
+
+class RunCommand(Command):
+    """Show help for commands"""
+    name = 'run'
+    usage = rwt.commands.help_doc
```

Comment:
I've pushed a new commit with docs for the pip run command. I've tested that these docs render properly and without warnings and provide a suitable background within the pip docs. I feel good about the documentation story now.